packages/components/src/ResourcePicker/Resource/Resource.scss

```diff
@@ -23,6 +25,22 @@
 		}
 	}
 
+	.flags-container {
+		margin-right: $padding-normal;
+		opacity: 0;
+
+		> svg {
+			margin-left: $padding-small;
+
+			&[name='talend-star'] {
+				color: $lightning-yellow;
+			}
+			&[name='talend-badge'] {
```

Comment:
Be less precise by using CSS class on top of this file (in order to be modified if needed)